### DISCUSSION TITLE
Clear Cookies

### DISCUSSION THREAD

**Discussion by orimoloyeb** [thunderclient/thunder-client-support#1090] [Q&A]
Hi

I have a collection. Some of the requests in the collection returns with response cookies. 
For example; in the collection there are three requests A, B, and C. When I run the collection, `A` sets a response cookie. How can I either ignore/clear the cookies set by `A` before `B` or `C` is executed when it is run as collection ? Im thinking I can use a pre-request script. However, I don't know what function to call on `tc`.

This is not a problem when not running as a collection because I am able to use the `Clear All` button to remove cookies when running the tests individually.

@rangav

**Comment by rangav** [comment]
Thanks @orimoloyeb for the feedback, will add to roadmap.

There is a similar feature request for this, Please add any further comments there.
https://github.com/rangav/thunder-client-support/issues/1091

**Comment by flooyo** [comment]
Hi @orimoloyeb,
maybe in the meantime a workaround would be (if you are aware of the cookienames) to set the cookies empty in the following requests. At least that's how I use it atm to run my collection with more iterations and without getting weird responses because there is still a value sent with that cookie.

The entry in the Header (and the variable holds my cookiename):
![image](https://user-images.githubusercontent.com/33790636/228804036-ab39eba4-aa9d-4070-8dcf-3026af0112ed.png)
